avocado_vt/plugins/vt_bootstrap.py

```diff
@@ -111,12 +112,14 @@ def run(self, args):
             else:
                 logging.error('Bootstrap command failed')
                 logging.error('Command: %s', ce.command)
-                if ce.result.stderr_text:
+                stderr = results_stderr_52lts(ce)
```

Comment:
shouldn't this be `stderr = results_stderr_52lts(ce.result)` ? If so, the line 119 needs correction as well.

Reply:
Sure, is that it or have you spotted other issues as well?

Reply:
no just that one, I would write be it that case.